plugins/modules/zos_job_output.py

```diff
@@ -175,6 +185,37 @@
                "         6 //SYSUT2   DD SYSOUT=*                                                          ",
                "         7 //                                                                              "
              ]
+    job_class:
+      description:
+        Job class for this job.
+      type: str
+      sample: A
+    svc_class:
+      description:
+        Service class for this job.
+      type: str
+      sample: C
+    priority:
+      description:
+        A numeric indicator of the job priority assigned through JES.
+      type: int
+      sample: 4
+    asid:
+      description:
+        The address Space Identifier (ASID) that is a unique descriptor for the job address space.
+        Zero if not active.
+      type: int
+      sample: 0
+    queue_position:
+      description:
+        The position within the job queue where the jobs resides.
+      type: int
+      sample: 3
+    program_name:
```

Comment:
Documenting `program_name` is a bit of problem for 1.7.0-beta.1 because the minimum ZOAU will be 1.2.3 and this will not appear in the response as our code will check the version of ZOAU but yet this is documented and misleading. This is why we generally don't expose no function unless that new function is in the minimum dependency requirement, eg the minimum for 1.7.0-beta.1 is 1.2.3 zoau, where for 1.8.x it will be zoau 1.2.4. 

Our options are:
1) Remove/disable program name till 1.8.x
2) Add a `returned: xxxxx` , generally its `returned: always` or similar, but we have done some stretch that norm to things like " returned: when ICKDSF program is run." so I suppose we could write something like: `returned: when ZOAU is 1.2.4 or later"
3) Write it into the description:
```
The name of the program found in the job's last completed step found in the PGM parameter.
Returned when Z Open Automation Utilities (ZOAU) is 1.2.4 or later. 
```

**This applies to `zos_job_submit` ,`zos_job_query` and zos_job_output`**

Reply:
I've added the 3/description line to all 3 zos_job modules.  We are a bit inconsistent with the returned: option in the return values field, so the description was easier to put in.